src/app/booking/OOSPlannerSection.js

```diff
@@ -260,6 +384,7 @@ function FamilyBookings() {
   const handlePreviosWeek = () => setSelectedDate(selectedDate.subtract(7, "day"));
 
   return (
+    !(isLoadingChildren || isLoadingBookings) && (
```

Comment:
It is a bit complex to read, as I need to understand negation with parenthesis.
Simpler: `!isLoadingChildren && !isLoadingBookings && ...`